logic/peers.js

```diff
@@ -254,6 +254,24 @@ Peers.prototype.list = function(normalize) {
 	);
 };
 
+/**
+ * Returns a random list of connected peers.
+ *
+ * @param {Object} [options] - Optional
+ * @param {number} [options.limit=null] - Maximum number of peers to get; defaults to all
+ * @returns {peer[]} List of peers
+ */
+Peers.prototype.listRandomConnected = function(options) {
+	options = options || {};
+	const peerList = Object.keys(self.peersManager.peers)
+		.map(key => self.create(self.peersManager.getByAddress(key)))
```

Comment:
`self.create` is not needed, as every peer that comes from `self.peersManager.peers` is alredy a `Peer` instance.